source/write/insert.txt

```diff
@@ -0,0 +1,171 @@
+
+To learn more about document structure and rules, see the
+:manual:`Documents </core/document>` guide in the {+mdb-server+} manual.
+
+Insert One Document
+-------------------
+
+To add a single document to a MongoDB collection, call the ``MongoDB\Collection::insertOne()``
+method and pass the document you want to add.
+
+The following example inserts a document into the ``restaurants`` collection:
+
+.. literalinclude:: /includes/write/insert.php
+    :language: php
+    :dedent:
+    :start-after: start-insert-one
+    :end-before: end-insert-one
+
+Insert Multiple Documents
+-------------------------
+
+To add multiple documents to a MongoDB collection, call the ``MongoDB\Collection::insertMany()``
+method and pass an array that contains the documents you want to add.
+
+The following example inserts two documents into the ``restaurants`` collection:
+
+.. literalinclude:: /includes/write/insert.php
+    :language: php
+    :dedent:
+    :start-after: start-insert-many
+    :end-before: end-insert-many
+
+Modify Insert Behavior
+----------------------
+
+You can modify the behavior of the ``MongoDB\Collection::insertOne()`` and
+``MongoDB\Collection::insertMany()`` methods by passing an array that specifies
+option values as a parameter. The following table describes some options
+you can set in the array:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Field
+     - Description
+
+   * - ``bypassDocumentValidation``
+     - | If set to ``true``, allows the write operation to opt out of
+         :manual:`document-level validation </core/schema-validation>`. 
+       | Defaults to ``false``.
+       | **Type**: ``bool``
+
+   * - ``writeConcern``
+     - | Sets the write concern for the operation.
+       | Defaults to the write concern of the namespace.
+       | **Type**: ``MongoDB\Driver\WriteConcern``
+
+   * - ``ordered``
+     - | If set to ``true``, the operation stops inserting documents when one insert
+         fails. If ``false``, the operation continues to insert the remaining documents
+         when one insert fails. You cannot pass this option to the ``insertOne()`` method.
+       | Defaults to ``true``.
+       | **Type**: ``bool``
+
+   * - ``comment``
+     - | A comment to attach to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         {+mdb-server+} manual.
+       | **Type**: any valid BSON type
+       
+Example
+~~~~~~~
+
+The following code uses the ``insertMany()`` method to insert three new
+documents into a collection. Because the ``bypassDocumentValidation`` field
+is set to ``true`` in an options array, this
+insert operation bypasses document-level validation:
+
+.. literalinclude:: /includes/write/insert.php
+    :language: php
+    :dedent:
+    :start-after: start-modify
+    :end-before: end-modify
+
+Additional Information
+----------------------
+
+.. TODO:
```

Comment:
Will this be addressed in a follow-up ticket?

Reply:
Yes, there will be a final cleanup ticket that resolves all the TODOs